### PR TITLE
Navigation Links

### PR DESCRIPTION
**Pull request by gkojunior**
Created a nav bar to link to the different pages/ routes

### CODE REVIEW

File: src/modules/home/components/LoginButton.js [hunk 1]
```diff
@@ -2,16 +2,17 @@ import React from 'react'
...
+import { Link } from 'react-router-dom'
 
 export const LoginButton = () => {
     const navigate = useNavigate()
     const onClick = () => navigate(loginPath)
 
     return (
         <div>
-            <button className={styles.loginButton} onClick={onClick}>
+            <Link className={styles.loginButton} onClick={onClick}>
```
Comment (Leigh-K) [inline src/modules/home/components/LoginButton.js]: instead of onClick use the `to` pro and pass it the loginPath route

Comment (Leigh-K) [reply src/modules/home/components/LoginButton.js:13]: wait do we even use either of these components anymore?

File: src/modules/home/components/LogoutButton.js [hunk 2]
```diff
@@ -15,9 +16,9 @@ export const LogoutButton = () => {
 
     return (
         <div>
-            <button className={styles.logoutButton} onClick={onClick}>
+            <Link className={styles.logoutButton} onClick={onClick}>
```
Comment (Leigh-K) [inline src/modules/home/components/LogoutButton.js]: can you use the `to`1 prop for navigation and the onClick just for logging out?

**Comment by AmyJenSandoval** [comment]
Note: might need to merge main since I think I only merged into main today and not dev

**Pull request merged**